### PR TITLE
prox/rxd: check seq_no for wrapping

### PR DESCRIPTION
**Pull request by aingerson**
Signed-off-by: aingerson <alexia.ingerson@intel.com>

### CODE REVIEW

File: prov/rxd/src/rxd.h [hunk 1]
```diff
@@ -246,6 +246,9 @@ static inline uint32_t rxd_flags(uint64_t fi_flags)
 	return rxd_flags;
 }
 
+#define rxd_after_eq(a,b)	((long)((a) - (b)) >= 0)
+#define rxd_before(a,b)		((long)((a) - (b)) < 0)
```
Comment (shefty) [inline prov/rxd/src/rxd.h]: Can you move these to common code?

Comment (aingerson) [reply prov/rxd/src/rxd.h]: Sure. Where? include/ofi_util.h?
Also, do you want me to include the set of calls (ie ofi_after, ofi_before, ofi_after_eq, ofi_before_eq) or just the two I used?

Comment (shefty) [reply prov/rxd/src/rxd.h]: ofi_util.h seems fine for now.  Just the 2 calls you used.

Comment (aingerson) [reply prov/rxd/src/rxd.h]: updated

**Comment by aingerson** [comment]
Repushed and everything back to normal except for AWS - ignoring and merging

**Pull request merged**